cfssl.tf

```diff
@@ -98,6 +76,27 @@ resource "aws_volume_attachment" "cfssl-data" {
   device_name = "/dev/xvdf"
   volume_id   = "${aws_ebs_volume.cfssl-data.*.id[count.index]}"
   instance_id = "${aws_instance.cfssl.*.id[count.index]}"
+
+  provisioner "remote-exec" {
+    when = "destroy"
+
+    inline = [
+      "sudo systemctl stop cfssl.service",
+      "sudo umount -d /dev/xvdf",
+      "sudo reboot",
```

Comment:
The above 2 commands were not enough to get rid of all the dependencies and aws was still failing on volume detach without restarting the instance first.

Reply:
Why `reboot`, you want a new instance, something like `halt` or `shutdown` would make more sense?